src/types/ImageStudio.ts

```diff
@@ -1,24 +1,25 @@
+import { SDKImage } from './SdkImage';
+
 /**
+ * @deprecated
```

Comment:
the readme has been updated, however its a good idea - do you have an example of how you would do this please?

Reply:
I'm not familiar enough with the changes in the PR to offer any specific examples, but can offer the more general advice of something like `@deprecated Use Foo instead`.

Here's an example from the node `fs.exists()` function:

```typescript
     * used directly, for example when its existence is a signal from another
     * process.
     * @since v0.0.2
     * @deprecated Since v1.0.0 - Use {@link stat} or {@link access} instead.
     */
    export function exists(path: PathLike, callback: (exists: boolean) => void): void;
```

Which then shows up in a IDE like so:
![image](https://github.com/user-attachments/assets/7555402f-6f39-42e1-90f2-13e857dddd45)

Reply:
This PR had no other significant changes required, so we will save these for another time so this can be merged without re-review.